### PR TITLE
v1.10

### PR DESCRIPTION
**Pull request by daler**
- Refactored testing into `test/lcdb-wf-test` script, which makes it much easier to run local tests
- Raise exception if `include`ed reference config does not exist (#368)
- Require `layout` column when using SRA sampletable (#367)
- Ensure all libraries are loaded before configuring parallelization (#365)
- All `lcdbwf` R functions use the `:::` namespace lookup
- R functions and args follow style favoring `_` rather than `.` in var names
- Remove redundant libsizes table
- Ensure .bai exists for dups-marked ChIP-seq bams (#351)
- Refactor snakefiles to use `resources:` directive, and correspondingly stop using the (long-deprecated) `--clusterconfig` mechanism. Resources have been copied over from the clusterconfig. Use `LCDBWF_SNAKEMAKE_PROFILE` env var to point to a central profile directory
- Lots of updates for CI and making everything work with mamba
- Refactored deploy script to rely on distutils rather than rsync exclude (which appears broken in recent versions)
- Support specification (or not) of orgdb for getting annotation db (useful for genomes with no or limited annotations)
- New function `lcdbwf:::lfc_scatter` for comparing multiple DESeq2 runs
- Updates and fixes to `gene-patterns.Rmd` 
- remove clusterconfig.yaml files and cluster_specific.py
- update docs

### CODE REVIEW

File: workflows/rnaseq/Snakefile [hunk 1]
```diff
@@ -138,7 +138,7 @@ if 'Run' in c.sampletable.columns and sum(c.sampletable['Run'].str.startswith('S
             sampletable=_st,
             # limit = 100000, # [TEST SETTINGS]
         resources:
-            mem_mb=1024,
+            mem_mb=autobump(1024),
```
Comment (daler) [inline workflows/rnaseq/Snakefile:141]: this should probably be changed to `disk_mb` for larger fastq files

File: include/WRAPPER_SLURM [hunk 2]
```diff
@@ -7,6 +7,22 @@
 # make logdir
 if [[ ! -e logs ]]; then mkdir -p logs; fi
 
+# use SNAKEMAKE_PROFILE if one is set in the environment variables
+if [ -z "$LCDBWF_SNAKE_PROFILE" ]; then
```
Comment (daler) [inline include/WRAPPER_SLURM:11]: let's change to `LCDBWF_SNAKEMAKE_PROFILE` just to be consistent with the original name

File: workflows/rnaseq/Snakefile (hunk 1 above)
Comment (esnaultc) [reply workflows/rnaseq/Snakefile]: done.

File: include/WRAPPER_SLURM (hunk 2 above)
Comment (esnaultc) [reply include/WRAPPER_SLURM]: changed everywhere

**Pull request merged**